x-pack/plugins/security_solution/public/resolver/view/panels/use_format_date.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment-timezone';
+import { useUiSetting } from '../../../../../../../src/plugins/kibana_react/public';
+import { formatter, invalidDateText } from './panel_content_utilities';
+
+export function useDateFormatSetting(
```

Comment:
This hook returns a formatted date so maybe this name could be `useFormattedDate`.

Reply:
lol, just renamed it. Saw I missed a few spots. Was confused why the ticket only named the event-detail, but it's all timestamps in the panel. Anyways will be updated shortly